systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -864,6 +865,8 @@ void testDynamicallySetMM2LoggingLevels() throws InterruptedException {
         KafkaResource.kafkaEphemeral(CLUSTER_NAME + "-target", 3).done();
         KafkaClientsResource.deployKafkaClients(false, KAFKA_CLIENTS_NAME).done();
         KafkaMirrorMaker2Resource.kafkaMirrorMaker2(CLUSTER_NAME, CLUSTER_NAME + "-target", CLUSTER_NAME + "-source", 1, false).done();
+        String kafkaMM2PodName = kubeClient().listPodsByPrefixInName(CLUSTER_NAME + "-" + Constants.MIRROR_MAKER2).get(0).getMetadata().getName();
```

Comment:
```suggestion
        String kafkaMM2PodName = KafkaMirrorMaker2Resource.kafkaMirrorMaker2Client().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getMetadata().getName();
```

Reply:
Thanks, I will update it right away.

Reply:
Sorry, my bad :smile: I meant this:
```suggestion
        String kafkaMM2PodName = kubeClient().listPods(Labels.STRIMZI_KIND_LABEL, KafkaMirrorMaker2.RESOURCE_KIND).get(0).getMetadata().getName();
```